src/components/general/Dropdown/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React, {
+    useState,
+    useRef,
+    useCallback,
+    useEffect,
+    MutableRefObject,
+    ReactNode,
+    FC,
+} from 'react';
+import {
+    TextInput,
+    DropdownArrow,
+    Menu,
+    useClickOutsideOverlayPortal,
+    useOverlayContainer,
+    useRelativePositioning,
+} from '@equinor/fusion-components';
+import styles from './styles.less';
+import { useFusionContext, FusionContext } from '@equinor/fusion';
+import { createPortal } from 'react-dom';
+import { Router } from 'react-router';
+
+type DropdownSelections = {
+    title: string;
+    key: string;
+};
+
+type DropdownProps = {
+    label?: string;
+    selections: DropdownSelections[];
+    onSelect?: (item: DropdownSelections) => void;
+    selected?: string;
+};
+
+const Dropdown: FC<DropdownProps> = ({ selections, label, onSelect, selected }) => {
+    const [open, setOpen] = useState(false);
+    const [inputValue, setInputValue] = useState('');
+    const [dropdownSelections, setDropdownSelections] = useState(selections);
```

Comment:
Should we have a useEffect on the selections prop incase the selections changes? E.g. the possible options is fetched from an API and not available in the first render. And also: Should the selections prop be renamed to "options"? And the type to DropdownOption instead of DropdownSelections?